server/app/models/va/VotingPlan.js

```diff
@@ -0,0 +1,41 @@
+// VotingPlan node definition
+//
+// Represents a voting plan, or a link that potentially leads to the creation
+// of a voting plan; related to the voter who is making the plan and the
+// canvasser who provided the link.
+//
+// This models the situation where an Ambassador canvasses a Tripler as:
+// provider: Ambassador -[:PROVIDES_LINK]-> VotingPlan -[:FOR_VOTER]-> voter: Tripler
+//
+// TODO(ping): This should also be used to model a Tripler canvassing a Triplee:
+// provider: Tripler -[:PROVIDES_LINK]-> VotingPlan -[:FOR_VOTER]-> voter: Triplee
+
+module.exports = {
+  id: { type: "uuid", primary: true },
+
+  // BallotReady provides a "ballot_id" field (see the "User Data Dictionary").
+  // It is uniquely generated each time a user begins their BallotReady flow,
+  // so it can also serve as a unique ID for a voting plan created by a voter.
+  ballot_id: "string",  // constrained unique
+
+  create_time: { type: "number", required: true },
+  last_send_time: "number",  // last time we texted this link to the voter
+
+  link_code: { type: "string", required: true },  // constrained unique
+  prefill_params: "string",  // stringified JSON
+
+  link_provider: {
+    type: "relationship",
+    direction: "in",
+    relationship: "PROVIDES_LINK",
+    target: "Ambassador",
+    cascade: "detach",
+  },
+  voter: {
+    type: "relationship",
+    direction: "out",
+    relationship: "FOR_VOTER",
+    target: "Tripler",
+    cascade: "detach",
+  }
```

Comment:
No need — the LinkCodeTemplate is just bookkeeping for generating the link codes.  There's no actual connection to the VotingPlan after that. (It would be like having a relationship with the whole roll of "Take a Number" tickets at the DMV; you only care about the ticket you took, not the roll of paper it came from.)